libs/ledger-live-common/src/bridge/crypto-assets/index.ts

```diff
@@ -0,0 +1,59 @@
+import { LiveConfig } from "@ledgerhq/live-config/lib/LiveConfig";
+import { CryptoAssetsStore, ERC20Token, SPLToken } from "./type";
+import * as legacy from "@ledgerhq/cryptoassets/tokens";
+import { TokenCurrency } from "@ledgerhq/types-cryptoassets";
+
+let cryptoAssetsStoreRef: CryptoAssetsStore | undefined = undefined;
+
+export function setCryptoAssetsStore(store: CryptoAssetsStore) {
+  cryptoAssetsStoreRef = store;
+}
+
+const legacyStore: CryptoAssetsStore = {
+  findTokenByAddress: legacy.findTokenByAddress,
+  getTokenById: legacy.getTokenById,
+  addTokens: legacy.addTokens,
+  convertERC20: convertERC20,
+  findTokenById: legacy.findTokenById,
+  findTokenByAddressInCurrency: legacy.findTokenByAddressInCurrency,
+  findTokenByTicker: legacy.findTokenByTicker,
+  convertSplTokens: convertSplTokens,
+};
+
+export function getCryptoAssetsStore(): CryptoAssetsStore | undefined {
+  const featureEnabled = LiveConfig.getValueByKey("feature_cal_lazy_loading");
+  if (!featureEnabled) {
+    return legacyStore;
+  }
+
+  if (!cryptoAssetsStoreRef) {
+    throw new Error("CryptoAssetsStore is not set. Please call setCryptoAssetsStore first.");
+  }
+
+  return cryptoAssetsStoreRef;
+}
+
+export function convertERC20(token: ERC20Token): TokenCurrency | undefined {
```

Comment:
Those functions (`convertERC20` and `convertSplTokens`) should be useless.
If I am not mistaken, they are needed for the conversion of "CAL" type token into "LedgerLive CryptoAsset" type token.
And this conversion is done only during the "preload" phase of `coin-evm` and `coin-solana`.
This phase is planned to be removed because it is redundant with the CALService calls.